common/redirect.ts

```diff
@@ -35,8 +47,10 @@ export function redirectHandlers(lambdas: lambda.Function[], addLayers: boolean)
     cfnFunction.handler = handler;
   });
 }
-
-function getDDHandler(lam: lambda.Function, addLayers: boolean) {
+/*
+ * Unchanged aside from parameter type
+ */
+function getDDHandler(lam: ILambdaFunction, addLayers: boolean) {
```

Comment:
Unchanged aside from `lambdas` parameter type (from aws import to LambdaFunction interface)